src/components/common/YourCompany/CompanyName/index.ts

```diff
@@ -1,4 +0,0 @@
-export { default as CorrectCompanyName } from './CorrectCompanyName.vue'
-export { default as CorrectNameOptions } from './CorrectNameOptions.vue'
-export { default as CorrectNameRequest } from './CorrectNameRequest.vue'
-export { default as CorrectNameToNumber } from './CorrectNameToNumber.vue'
```

Comment:
Don't need this as the only actual export was CorrectNameOptions (now called CorrectName), and the others were only used by CorrectName.